pkg/systemd/overview-cards/cryptoPolicies.jsx

```diff
@@ -0,0 +1,220 @@
+                                .then(() => close());
+                    }
+                })
+                .catch(error => setError(error));
+    };
+
+    const policies = Object.keys(cryptopolicies).map(policy => ({
+        name: policy,
+        title: displayProfileText(policy),
+        description: cryptopolicies[policy],
+        active: policy === currentCryptoPolicy,
+        recommended: false,
+    }));
+
+    // Custom profile
+    if (!(currentCryptoPolicy in cryptopolicies)) {
+        policies.push({
+            name: currentCryptoPolicy,
+            title: displayProfileText(currentCryptoPolicy),
+            description: _("Custom crypto policy"),
+            active: true,
+            recommended: false,
+        });
+    }
+
+    const help = (
+        <Popover
+            id="crypto-policies-help"
+            bodyContent={
+                <div>
+                    {_("Crypto Policies is a system component that configures the core cryptographic subsystems, covering the TLS, IPSec, SSH, DNSSec, and Kerberos protocols.")}
+                </div>
+            }
+            footerContent={
+                <Button component='a'
+                        rel="noopener noreferrer" target="_blank"
+                        variant='link'
+                        isInline
+                        icon={<ExternalLinkSquareAltIcon />} iconPosition="right"
+                        href="https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/using-the-system-wide-cryptographic-policies_security-hardening">
+                    {_("Learn more")}
+                </Button>
+            }
+        >
+            <Button variant="plain" aria-label={_("Help")}>
+                <HelpIcon />
+            </Button>
+        </Popover>
+    );
+
+    return (
+        <Modal position="top" variant="medium"
+               isOpen
+               help={help}
+               onClose={close}
+               id="crypto-policy-dialog"
+               title={_("Change crypto policy")}
+               footer={
+                   <>
+                       {error && <ModalError dialogError={typeof error == 'string' ? error : error.message} />}
+                       <Button id="crypto-policy-apply-reboot" variant='primary' onClick={() => setPolicy(true)}>
+                           {_("Apply and reboot")}
+                       </Button>
+                       <Button id="crypto-policy-apply-reboot-later" variant='secondary' onClick={() => setPolicy(false)}>
+                           {_("Apply only")}
+                       </Button>
+                       <Button variant='link' onClick={close}>
+                           {_("Cancel")}
+                       </Button>
+                   </>
+               }
+        >
+            {currentCryptoPolicy && <TunedDialogBody active_profile={currentCryptoPolicy}
```

Comment:
It feels rather odd (and looks like a copy&paste error) to use `TunedDialogBody` here. Could we split that out into a separate component and use it from both places? That's fine for me as a follow-up, though.